### PR TITLE
Feature/add navbar

### PR DESCRIPTION
**Pull request by henrikav**

### CODE REVIEW

File: tailwind.config.ts [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+		extend: {
+			backgroundImage: {
+				"gradient-radial": "radial-gradient(var(--tw-gradient-stops))",
+				"gradient-conic":
+					"conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))",
+			},
+			fontFamily: {
+				sans: ["var(--font-inter)"],
+			},
+			theme: {
```
Comment (dymbe) [inline tailwind.config.ts:21]: Did you export this theme from Figma?

File: app/components/MissionSection.tsx [hunk 2]
```diff
@@ -31,10 +30,7 @@ export default function MissionSection() {
 					>
 						Local Tech Innovation
 					</Typography>
-					<Typography
-						variant="lead"
-						className="text-[#0C1F14] lg:w-[69%] text-[22px]"
-					>
+					<Typography className="text-[#0C1F14] lg:w-[69%] text-[22px]">
```
Comment (dymbe) [inline app/components/MissionSection.tsx:33]: The color is already defined in the theme, so you should be able to use it with `text-secondary-10`

File: app/components/FeatureCard.tsx [hunk 3]
```diff
@@ -20,13 +21,12 @@ export default function FeatureCard({
 	return (
 		<Card
 			style={{ background: promoted ? "#D1E8D5" : "#F2F4EF" }}
```
Comment (dymbe) [inline app/components/FeatureCard.tsx:23]: #D1E8D5 is defined in theme, and #F2F4EF is not just a one-off, so I would consider adding it to the theme as well

**Review by dymbe — CHANGES_REQUESTED** [review]

File: tailwind.config.ts (hunk 1 above)
Comment (henrikav) [reply tailwind.config.ts:21]: Exported as json from Figma. Already exists in flood-frontend and developer-portal.

File: app/components/MissionSection.tsx (hunk 2 above)
Comment (henrikav) [reply app/components/MissionSection.tsx:33]: This content is not relevant for this PR. Resolving as this will be fixed in an upcoming PR

**Review by dymbe — APPROVED** [review]

**Pull request merged**